.github/workflows/release-tagger.yaml

```diff
@@ -0,0 +1,23 @@
+name: Tag Release
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  TagRelease:
+    permissions:
+      contents: write
+    name: Create Release Tag
+    runs-on: ubuntu-22.04
+    if: github.event.pull_request.merged == true && startsWith( github.event.pull_request.head.ref, 'release-' )
```

Comment:
@hawksight See here ^^ It will only actually run IF its merged and was `release-xxx`